server/schema/ProjectResolvers.ts

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable no-underscore-dangle */
+import { IResolvers } from '@graphql-tools/utils';
+import randomString from 'randomstring';
+import { Document } from 'mongoose';
+
+import UserDB from '../models/User';
+import QueryDB from '../models/Query';
+import ProjectDB from '../models/Project';
+
+function isBucketLimiter(rateLimiter: RateLimiterType): rateLimiter is BucketType {
+    return ['TOKEN_BUCKET', 'LEAKY_BUCKET'].includes(rateLimiter as BucketType);
+}
+
+function isWindowLimiter(rateLimiter: RateLimiterType): rateLimiter is WindowType {
+    return ['FIXED_WINDOW', 'SLIDING_WINDOW_LOG', 'SLIDING_WINDOW_COUNTER'].includes(
+        rateLimiter as WindowType
+    );
+}
+
```

Comment:
Expect a ton of conflicts in here. Dewey is overhauling the resolvers because of ts compile problems when returning mongo documents to graphQL types